azure/bastion/outputs.tf

```
@@ -15,5 +15,5 @@ output "name" {
 
 output "resource_group_name" {
   description = "Name of the resource group."
-  value       = local.resource_group_name
-}
+  value       = var.resource_group_name
```

Unsure why this is here or if it's from something older but would suggest at least tying this back to a resource otherwise this is literally going from input->output with no dependency or anything tying it in.

If it has to be done,
```suggestion
  value       = azurerm_bastion.main.resource_group_name
```